workflows/TW-phoSim-r3/helpers/updateGit.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+taskName=TW-phoSim-r3
+pipelineRoot=/nfs/farm/g/desc/u1/Pipeline-tasks/${taskName}
+
+GLOBIGNORE="*"
+src="${pipelineRoot}/config/"
+dst=${pipelineRoot}/Twinkles/workflows/${taskName}/
+
+echo src "${src}"
+echo dst ${dst}
+
+cmd="rsync -avuzb  --exclude ""*~"" ${src} ${dst}"
```

Comment:
I think the copying here is going the opposite direction of what I would expect.  The files in the git repository should be your reference versions and they should be copied __to__ the execution area, not the other way around.  In fact, if I were doing this, at least for development, I would make symlinks in execution area that point back to the files in the repository, that way any edits I make in the repo are automatically available when I try to run the code.

Also, it's not clear to me that everything that ends up in the execution area should be copied back to the repository.  I'm thinking, in particular, of `twinkles_visits.txt` which would be created by running `get_twinkles_visits.py`.  Since that's generated by our code, it's really a data product of the execution of the data generation pipeline and should be put into the Data Catalog with metadata identifying it as being used in Run3, for example.